### PR TITLE
Fix Issue  #160, #162, #171, #182

### LINKED ISSUE #160: [PE-D][Tester C] Documentation error for Add function
### Documentation error for Add function

Input command: `add n/John Doe a/John Street, Block 123, #01-01 appt/12-12-2023 08:30`

Actual output: `Invalid command format! 
add: Adds a person to the address book. Parameters: n/NAME p/PHONE e/EMAIL a/ADDRESS [t/TAG]...
Example: add n/John Doe p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 t/friends t/owesMoney`

However, user guide mentions that:
"Precise expected outputs on failure:
If a required parameter is missing (e.g., NAME, EMAIL), an error message should specify which parameter is missing."

As a user, I expected that the actual output will say that the input command is missing email and phone but it was not the case.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: nicrandomlee/ped#3

### LINKED ISSUE #162: [PE-D][Tester B] Invalid example given for add command(invalid command format output)
I copied the example for add command when i just type "add" in the input. It shows invalid command format, instead of adding the person

-------------
Labels: `type.FunctionalityBug` `severity.Low`
original: Kailash201/ped#3

### LINKED ISSUE #171: [PE-D][Tester B] edit function not working
edit function not working when i want change the occupation to police-man. according to your ug, o/ takes in a valid string input

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: Kailash201/ped#4

### LINKED ISSUE #182: [PE-D][Tester E] Add feature in UG missing appt field
In command summary, appt is missing from the format.

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: wnchan/ped#4

### PR DESCRIPTION
**Pull request by bhnuka**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -167,19 +167,19 @@ Example: riskprofile 1 res/a,e,b,d,c,a,d,e
 Adds a new entry to their address book for financial advisors, including personal details such as name, address, occupation, phone number and email.
 
 #### Format:
-* `add n/NAME p/PHONE_NUMBER e/EMAIL o/OCCUPATION a/ADDRESS [t/TAG] [appt/APPOINTMENT_DATE]`
+* `add n/NAME p/PHONE_NUMBER e/EMAIL o/OCCUPATION a/ADDRESS [t/TAG]… [appt/APPOINTMENT_DATE]`
```
Comment (syamfarh) [inline docs/UserGuide.md:170]: I think we need to standardize parameter `APPOINTMENT_DATE`.
I notice some part of the UG use `APPOINTMENTDATE` while other parts use `APPOINTMENT_DATE` or `APPOINTMENT DATE`

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 2]
```diff
@@ -25,15 +27,19 @@ public class AddCommand extends Command {
             + PREFIX_NAME + "NAME "
             + PREFIX_PHONE + "PHONE "
             + PREFIX_EMAIL + "EMAIL "
+            + PREFIX_OCCUPATION + "OCCUPATION "
```
Comment (syamfarh) [inline src/main/java/seedu/address/logic/commands/AddCommand.java:30]: for issue #162

File: docs/UserGuide.md [hunk 3]
```diff
@@ -903,23 +903,23 @@ If your changes to the data file makes its format invalid, FAPro will discard al
...
-| **Risk Profile**    | `riskprofile 3 res/a,b,c,d,e,e,b,c`                                                                                                                                                      |
-| **Exit**            | `exit`                                                                                                                                                                                   |
+| Action              | Format, Examples                                                                                                                                                                                                                      |
+|---------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Add**             | `add n/NAME p/PHONE_NUMBER e/EMAIL o/OCCUPATION a/ADDRESS [t/TAG]… [appt/APPOINTMENT_DATE]` <br> e.g. `add n/James Ho p/22224444 e/jamesho@example.com o/SWE, a/123, Clementi Rd, 1234665 t/friend t/colleague appt/12-12-2023 08:30` |
```
Comment (syamfarh) [inline docs/UserGuide.md:908]: fixed for issue #182

File: docs/UserGuide.md [hunk 4]
```diff
@@ -204,7 +204,7 @@ where X are the details of the person added
 
 #### Precise expected outputs on failure:
 
-If a required parameter is missing (e.g., NAME, EMAIL), an error message should specify which parameter is missing.
+If a required parameter is missing (e.g., NAME, EMAIL)
```
Comment (syamfarh) [inline docs/UserGuide.md:207]: fixed for issue #160

File: docs/UserGuide.md (hunk 1 above)
Comment (syamfarh) [reply docs/UserGuide.md:170]: alright, in the meantime, I approve the PR first. I try to fix it in my PR instead!

File: docs/UserGuide.md [hunk 5]
```diff
@@ -167,19 +167,19 @@ Example: riskprofile 1 res/a,e,b,d,c,a,d,e
...
-* ADDRESS: Valid string address (8 College Ave West)
-* PHONE NUMBER: Valid string (Numbers only) (81234567)
-* EMAIL ADDRESS: Valid string and email address format (johnd@example.com)
-* OCCUPATION: Valid string occupation
-* TAG: Valid string
+* NAME: Must be alphanumeric characters only. Name must be unique. (John Doe)
+* ADDRESS: Can take any values except blank (8 College Ave West)
+* PHONE NUMBER: Numbers only. Must be at least 3 digits long. (81234567)
+* EMAIL ADDRESS: Valid email address format. Accepts any alphanumeric characters other than underscore (johnd@example.com)
+* OCCUPATION: Must be alphanumeric characters only
```
Comment (syamfarh) [inline docs/UserGuide.md:181]: fixed for issue #171

**Review by syamfarh — COMMENTED** [review]
LGTM, just a nitpick on standardising parameter

File: docs/UserGuide.md (hunk 1 above)
Comment (bhnuka) [reply docs/UserGuide.md:170]: I think when talking about the parameter, definitely not APPOINTMENT DATE. Maybe APPOINTMENT_DATE? I'll check with the group.

**Pull request merged**